FWCore/Framework/interface/stream/ProducingModuleAdaptor.h

```diff
@@ -137,11 +137,11 @@ namespace edm {
           const EventSetup c{ci,
                              static_cast<unsigned int>(Transition::EndRun),
                              this->consumer()->esGetTokenIndices(Transition::EndRun)};
+          MyGlobalRunSummary::globalEndRun(r, c, &rc, m_runSummaries[ri].get());
           if constexpr (T::HasAbility::kEndRunProducer) {
             MyEndRunProduce::produce(r, c, &rc, m_runSummaries[ri].get());
             this->commit(r);
           }
-          MyGlobalRunSummary::globalEndRun(r, c, &rc, m_runSummaries[ri].get());
```

Comment:
@Dr15Jones Are the stream EDFilters also covered by changes in `stream/ProducingModuleAdaptor.h`?

Reply:
It looks that way to me.

Reply:
Tests confirm.